### PR TITLE
 perf: redwood release support

### PR DESCRIPTION
**Pull request by luisfelipec95**
## Description

This PR adds support for the Open edx Redwood release.

## Testing instructions

1. Create an environment with Redwood release, you can use [Tutor](https://docs.tutor.overhang.io/index.html) or [TVM](https://github.com/eduNEXT/tvm)
2. Install the Django plugin eox-tagging, you can use steps in [Tutor documentation](https://docs.tutor.overhang.io/configuration.html#installing-extra-xblocks-and-requirements).
3. Run [test cases](https://docs.google.com/document/d/1VnjGF6Ri1Bvgbcd8AulM5jt68U7BmYJLVV2H5Bi5VOg/edit#heading=h.gcaxh6e3de55)

## Aditional information
https://edunext.atlassian.net/browse/AP-1015

### CODE REVIEW

File: .github/workflows/commitlint.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ jobs:
   commitlint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4.1.7
         with:
           fetch-depth: 0
-      - uses: wagoid/commitlint-github-action@v5
+      - uses: wagoid/commitlint-github-action@v6.0.1
```
[the remark this answers is not in the document]
Comment (luisfelipec95) [reply .github/workflows/commitlint.yml]: Done, thanks

**Comment by luisfelipec95** [comment]
The following tests were performed on stage https://redwood.releases.edunext.link/

**Comment by mariajgrimaldi** [comment]
@luisfelipec95: is the postman collection available for us to test as well? Could that be part of the cover letter?

**Review by mariajgrimaldi — APPROVED** [review]
LGTM! I'm only missing testing on a remote environment.

**Review by bra-i-am — APPROVED** [review]
I tested it with the collection you shared with us, and it works as expected!

**Pull request merged**